tests/Directory.Build.targets

```diff
@@ -8,6 +8,7 @@
     <VSTestRunSettingsFile Condition="'$(VSTestRunSettingsFile)' == ''">$(RunSettingsFilePath)</VSTestRunSettingsFile>
 
     <DeployRunSettingsFile Condition="'$(DeployRunSettingsFile)' == ''">true</DeployRunSettingsFile>
+    <XunitRunnerJson Condition="'$(XunitRunnerJson)' == '' and '$(ArchiveTests)' == 'true'">$(RepoRoot)tests\helix\xunit.runner.json</XunitRunnerJson>
```

Comment:
Could you add a comment above here to explain why we need a special one for Helix?

Reply:
Meaning adding something like "In Helix we want to disable test parallelization in order to not hit docker network limit issues bla bla.."